src/routes/auth.ts

```diff
@@ -0,0 +1,112 @@
+import express from 'express'
+import { ErrorTypes, SiweMessage } from 'siwe'
+import { validateSchema } from '../schema'
+import {
+  AuthRequestBody,
+  InvalidSiweParamsError,
+  NotImplementedError,
+} from '../types'
+import { asyncRoute } from './async-route'
+
+const MAX_EXPIRATION_TIME_MS = 4 * 60 * 60 * 1000 // 4 hours
+const VERSION = '1'
+const CHAIN_ID = 42220
```

Comment:
CHAIN_ID should likely be parameterizable somehow, to allow for deployments on Alfajores. Maybe it doesn't need to be directly configurable by the end-user, but can be set through the [authConfigOptions](https://github.com/fiatconnect/api-starter/blob/main/src/config.ts#L15).